### PR TITLE
Ttl token migration

### PR DESCRIPTION
**Pull request by alfrunes**
Migrated the `token` collection with TTL index on the `ExpireAt` field s.t. mongo automatically removes expired tokens.
The token model is migrated to use uuid types with bson compatible document embedding.

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ listen: :8080
 
     # JWT expiration in seconds ('exp' claim)
     # Defaults to: "604800" (one week)
-# jwt_exp_timeout: 604800
+jwt_exp_timeout: 60
```
Comment (tranchitella) [inline config.yaml]: @alfrunes is this on purpose?

File: tests/common.py [hunk 2]
```diff
@@ -51,64 +64,86 @@ def make_basic_auth(username, password):
...
 @pytest.fixture(scope="session")
 def mongo():
-    return MongoClient('mender-mongo:27017')
+    return MongoClient("mender-mongo:27017")
+
 
 def mongo_cleanup(mongo):
     dbs = mongo.database_names()
-    dbs = [d for d in dbs if d not in ['local', 'admin', 'config']]
+    dbs = [d for d in dbs if d not in ["local", "admin", "config"]]
```
Comment (tranchitella) [inline tests/common.py]: A lot of changes in this diff are due to the black substitution `" -> '`.
Not a big deal, indeed. Just one thing: maybe running black first, then changing the code in a separate commit makes the review easier? Don't know...

Comment (alfrunes) [reply tests/common.py]: ahh, sorry. I've set up my editor to run `black` formatter on save. If there are a lot of such changes I can remove them from this PR.

File: config.yaml (hunk 1 above)
Comment (alfrunes) [reply config.yaml]: Thanks for having my back. No, this was just for verifying that it works (and indeed it does :).

**Comment by alfrunes** [comment]
I don't know what's up with the CI pipeline, a lot of connection refused errors. It passed locally.

File: tests/common.py (hunk 2 above)
Comment (tranchitella) [reply tests/common.py]: It is okay, don't worry!

File: config.yaml (hunk 1 above)
Comment (tranchitella) [reply config.yaml]: Good!

**Pull request merged**